Directory.Build.props

```diff
@@ -0,0 +1,30 @@
+<Project>
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)obj\$(MSBuildProjectName)\</BaseIntermediateOutputPath>
+    <BaseOutputPath Condition=" '$(BaseOutputPath)' == '' ">$(MSBuildThisFileDirectory)bin\$(MSBuildProjectName)\</BaseOutputPath>
+
+    <GenerateDocumentationFile>true</GenerateDocumentationFile>
+    <LangVersion>7.3</LangVersion>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+
+    <IsPackable>false</IsPackable>
+    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
+    <PackageLicenseFile>LICENSE</PackageLicenseFile>
+    <Authors>neuecc</Authors>
+    <Owners>neuecc</Owners>
+    <Copyright>© Yoshifumi Kawai. All rights reserved.</Copyright>
+    <PackageProjectUrl>https://github.com/AArnott/MessagePack-CSharp</PackageProjectUrl>
```

Comment:
If merged, is PackageProjectUrl invalid?

Reply:
Ah, I'll fix that to point at yours. :)

Reply:
Brandylee24 is unauthorized